Makefile

```diff
@@ -32,7 +32,7 @@ provision:
 	nht float --master --no-destroy --skip-gtg --testapp ${TEST_APP} --vault
 
 run:
-	nht run --local --https
+	nht run --https
```

Comment:
I think you should be using `make run-local` instead so I'm not sure this should be removed.

Reply:
why?

Reply:
IDK

Reply:
Cargo culting